api/v1/v1_test.go

```diff
@@ -120,9 +125,9 @@ func TestEnd2End(t *testing.T) {
 		pushRefs, err := runEnd2EndJob(testCase.pullRefs, testCase.seedRefs)
 
 		if testCase.isCorrect {
-			assert.Nil(err, "should be no error")
+			assert.Nil(err, fmt.Sprintf("should be no error (%+v)", testCase))
 		} else {
-			assert.NotNil(err, "should be an error")
+			assert.NotNil(err, fmt.Sprintf("should be an error (%+v)", testCase))
```

Comment:
This is to have more observability on what happened during failed tests.

Reply:
Very nice!!!